lib/ruby_lsp/ruby_lsp_rails/code_lens.rb

```diff
@@ -31,13 +32,23 @@ module Rails
     # end
     # ````
     #
+    # # Example:
+    # ```ruby
+    # Run in terminal
+    # class AddFirstNameToUsers < ActiveRecord::Migration[7.1]
+    #   # ...
+    # end
+    # ````
+    #
     # The code lenses will be displayed above the class and above each test method.
     class CodeLens < ::RubyLsp::Listener
       extend T::Sig
       extend T::Generic
 
       ResponseType = type_member { { fixed: T::Array[::RubyLsp::Interface::CodeLens] } }
-      BASE_COMMAND = "bin/rails test"
+      MIGRATE_COMMAND = "bin/rails db:migrate"
+      TEST_COMMAND = "bin/rails test"
+      BASE_COMMAND = TEST_COMMAND # TODO: Deprecate?
```

Comment:
We can definitely remove this. This is the only class that should be using this constant.